changelog/README.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 11th November 2022
+
+* Added cursor pagination in [Get all products](../operations/products.md#get-all-products).
```

Comment:
```suggestion
* Added [Pagination](../guidelines/pagination.md) to [Get all products](../operations/products.md#get-all-products).
```
Let's keep it consistent

Reply:
@VBuben 
![image](https://user-images.githubusercontent.com/24482879/201298130-c19bb162-ffe4-4ab7-b49b-bfccfd5c6d4d.png)

I tried 😊

Reply:
Ok I totally missed 🤣  that but I think what I put there is better since it redirects you to the Pagination page 👍